src/Sylius/Bundle/ApiBundle/ApiPlatform/Metadata/MergingXmlExtractor.php

```diff
@@ -107,8 +107,8 @@ protected function extractPath(string $path): void
                     $resourceMetadata,
                 );
                 $this->properties[$resourceClass] = array_merge(
-                    $this->properties[$resourceClass],
-                    $resourceMetadata['properties'],
+                    $this->properties[$resourceClass] ?? [],
+                    $resourceMetadata['properties'] ?? [],
```

Comment:
null cannot be merged with []
Maybe adjusting \Sylius\Bundle\ApiBundle\ApiPlatform\Metadata\MergingXmlExtractor::buildResource properties to not default to null is better?